src/bus/lattice.rs

```diff
@@ -124,13 +135,62 @@ impl DistributedBus {
             backoffcount += 1;
         }
         let _ = self.publish_event(BusEvent::HostStopped(self.host_id.to_string()));
+        std::thread::sleep(Duration::from_millis(300));
```

Comment:
Is there a better way for us to know the host has stopped? What if it takes longer than 300 millis for that message to get delivered, etc.

Reply:
There are some ways to know, but often detecting them requires using the lattice feature when that feature might not be available during the test ;)